langkit/support/langkit_support-bump_ptr.adb

```diff
@@ -135,8 +135,17 @@ package body Langkit_Support.Bump_Ptr is
 
    package body Alloc is
 
-      function To_Pointer is new Ada.Unchecked_Conversion
-        (System.Address, Element_Access);
+      -----------
+      -- Alloc --
+      -----------
+
+      function Alloc (Pool : Bump_Ptr_Pool) return System.Address is
+      begin
+         return
+           Allocate
+             (Pool,
+              Align (Element_T'Max_Size_In_Storage_Elements, Pointer_Size));
+      end Alloc;
```

Comment:
This seems to duplicate the code of the other Alloc function. Could you make the other one call this one?